Skyflow/src/main/kotlin/Skyflow/utils/Utils.kt

```diff
@@ -485,5 +488,21 @@ public class Utils {
         fun currentMonth(): Int {
             return Calendar.getInstance().get(Calendar.MONTH) + 1
         }
+
+        fun fetchMetrics(): JSONObject {
+            val metrics = JSONObject()
+            try {
+                metrics.put(
+                    "sdk_name_version",
+                    "${BuildConfig.SDK_NAME} ${BuildConfig.SDK_VERSION}"
```

Comment:
it should be in this format
sdk_name_version: ${BuildConfig.SDK_NAME}@${BuildConfig.SDK_VERSION}